### DISCUSSION TITLE
music/bluebox/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#11] [General]
# music/bluebox/

http://shanenull.com/music/bluebox/

**Comment by shane0** [comment]
Fx bus https://youtu.be/j9YZ8EkwW2Q?si=tMqN_j6DUi9urTte

**Comment by shane0** [comment]
https://1010music.com/wp-content/uploads/2021/07/blueboxusermanual-1-1-2-a-with-highlights.pdf
